Multisig/UI/Safe Management/Add Safe/EnterUnstoppableNameViewController.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  EnterUnstoppableNameViewController.swift
+//  Multisig
+//
+//  Created by Johnny Good on 4/21/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class EnterUnstoppableNameViewController: UIViewController {
+    var onConfirm: () -> Void = { }
+    var manager = App.shared.blockchainDomainManager
+    var address: Address?
+
+    // generated "task" ID to work around the asynchronous ENS resolving API
+    private var currentResolutionTaskID: UUID?
+    private var confirmButton: UIBarButtonItem!
+    private var debounceTimer: Timer!
+    private let debounceDuration: TimeInterval = 0.250
+
+    @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet private weak var textField: GNOTextField!
+    @IBOutlet private weak var addressFoundStackView: UIStackView!
+    @IBOutlet private weak var foundHeaderLabel: UILabel!
+    @IBOutlet private weak var foundIdenticonView: UIImageView!
+    @IBOutlet private weak var foundAddressLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Enter Unstoppable Name"
+
+        confirmButton = UIBarButtonItem(title: "Confirm", style: .done, target: self, action: #selector(didTapConfirmButton))
+        confirmButton.isEnabled = false
+        navigationItem.rightBarButtonItem = confirmButton
+
+        textField.setPlaceholder("Enter Unstoppable name")
+        textField.textField.autocorrectionType = .no
+        textField.textField.autocapitalizationType = .none
+        textField.textField.keyboardType = .URL
+        textField.textField.delegate = self
+        textField.textField.becomeFirstResponder()
+
+        addressFoundStackView.isHidden = true
+        foundHeaderLabel.setStyle(.primary)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        trackEvent(.safeAddEns)
```

Comment:
We need to add `.safeAddUD` event. And also please assure that it is the same on Android.